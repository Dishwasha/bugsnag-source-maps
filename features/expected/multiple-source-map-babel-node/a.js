"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;

class A {
  constructor(b) {
    this.b = b;
  }

  hello() {
    this.b.hey();
    console.log('hello');
  }

}

exports.default = A;
//# sourceMappingURL=a.js.map
