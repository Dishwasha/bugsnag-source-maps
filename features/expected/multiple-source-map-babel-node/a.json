{
    "version": 3,
    "sources": [
        "src/lib/a.js"
    ],
    "names": [
        "A",
        "constructor",
        "b",
        "hello",
        "hey",
        "console",
        "log"
    ],
    "mappings": ";;;;;;;AAAe,MAAMA,CAAN,CAAQ;AACrBC,EAAAA,WAAW,CAACC,CAAD,EAAI;AACb,SAAKA,CAAL,GAASA,CAAT;AACD;;AAEDC,EAAAA,KAAK,GAAG;AACN,SAAKD,CAAL,CAAOE,GAAP;AAEAC,IAAAA,OAAO,CAACC,GAAR,CAAY,OAAZ;AACD;;AAToB",
    "sourcesContent": [
        "export default class A {\n  constructor(b) {\n    this.b = b\n  }\n\n  hello() {\n    this.b.hey()\n\n    console.log('hello')\n  }\n}\n"
    ],
    "file": "a.js"
}
